code/style.css

```diff
@@ -1,7 +1,347 @@
+/* styling for the whole page */
+* {margin:0;}
+
+body {
+  background-color: #271916;
+}
+
+/* styling for the header */
+#hero {
+  height: 1000px; 
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#hero video {
+  width: 100%;
+  height: 1000px;
+  position: absolute;
+  top: 0;
+  left:0;
+  z-index: 5;
```

Comment:
Why do we use z-index? I am not familiar with it but will do some research :)

Reply:
To secure that this element will be on top of the other that has a smaller z-index :) 
Yes, check it out, very useful thing https://www.w3schools.com/cssref/pr_pos_z-index.asp